### PR TITLE
Moraru no fire on base

### PR DESCRIPTION
**Pull request by RaduM84**
Extended "Fired" event handler for Players (initPlayerLocal.sqf) and Vehicles (respawnVehicleInitialized) to delete projectiles fired while within the "headquarters" marker.

### CODE REVIEW

**Comment by BojanOnTheInternet** [comment]
Not 100% sure what's going on with this commit history and the diff here, but shits fucked yo. Can you try and get rid of all these extra files and avoid the merge conflicts?

File: 7Cav-Alive-WFW.Altis/scripts/vehicle/fn_respawnVehicleInitialize.sqf [hunk 1]
```diff
@@ -44,3 +44,12 @@ if (!isNil "_vehicleInit") then
 };
 
 [_vehicle] spawn JB_RV_Monitor;
+
```
Comment (BojanOnTheInternet) [inline 7Cav-Alive-WFW.Altis/scripts/vehicle/fn_respawnVehicleInitialize.sqf:47]: You don't need to have the code in both folders, the CI will copy any scripts over to the other mission files. We only need to edit scripts in the main Altis mission folder. You can leave this in, it'll just get overwritten.

File: 7Cav-Alive-WFW.Altis/initPlayerLocal.sqf [hunk 2]
```diff
@@ -48,3 +48,12 @@ player addEventHandler ["HandleRating", { 0 }];
 
 CLIENT_InitPlayerLocalComplete = true;
 diag_log "initPlayerLocal end";
+
```
Comment (BojanOnTheInternet) [inline 7Cav-Alive-WFW.Altis/initPlayerLocal.sqf:51]: See comment re: duplicate files below

File: 7Cav-Alive.Altis/initPlayerLocal.sqf [hunk 3]
```diff
@@ -48,3 +48,12 @@ player addEventHandler ["HandleRating", { 0 }];
...
 diag_log "initPlayerLocal end";
+
+// No firing on base
+player addEventHandler ["Fired", {
+	if ((getPos (_this select 0)) inArea headquarters)  then	
+	{
+		deleteVehicle (_this select 6);
+		titleText ["Firing weapons and placing / throwing explosives at base is STRICTLY PROHIBITED!", "PLAIN", 3];
+	};
+}];
```
Comment (BojanOnTheInternet) [inline 7Cav-Alive.Altis/initPlayerLocal.sqf:59]: Move

```
CLIENT_InitPlayerLocalComplete = true;
diag_log "initPlayerLocal end";
```

to end of file

**Review by BojanOnTheInternet — CHANGES_REQUESTED** [review]

**Comment by BojanOnTheInternet** [comment]
Other than the above looks good to go 👌

**Comment by RaduM84** [comment]
I'll pull the newest branch and re-add my changes.

**Comment by davidcit646** [comment]
What're we doing with this?

**Comment by Jaidendude** [comment]
@RaduM84 Give us an update please, else I'll close the pull request for now for being stale

**Review by Jaidendude — CHANGES_REQUESTED** [review]
Please provide an update.

**Pull request closed without merging**